src/app/containers/Headline/index.stories.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import HeadlineContainer from './index';
+import { textBlock } from '../../models/blocks';
+
+storiesOf('HeadlineContainer', module).add('default', () => (
```

Comment:
Due to this component being in a different directory the HeadlineContainer shows nowhere near the Headline. I don't think there is a trivial way to fix this. We could do this instead: `storiesOf('Headline', module).add('container',` but it doesn't seem much better. Would need to do the same for all containers.